SU2_CFD/include/solvers/CMeshSolver.hpp

```diff
@@ -154,12 +154,6 @@ class CMeshSolver : public CFEASolver {
    */
   void SetBoundaryDisplacements(CGeometry *geometry, CNumerics *numerics, CConfig *config);
 
-  /*!
-   * \brief Set the value of the max residual and BGS residual.
-   * \param[in] val_iterlinsolver - Number of linear iterations.
-   */
-  void ComputeResidual_Multizone(CGeometry *geometry, CConfig *config);
```

Comment:
@rsanfer I changed CMeshSolver to use "BGS_k" instead of "Solution_Old" since the former is now allocated by CVariable whenever the problem is multizone, that allows the base implementation of 'ComputeResidual_Multizone' to be used. Do you foresee any problems?